Sorting/TQuick.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+
+namespace Sorting
+{
+    class TQuick : TSorting
+    {
+        public TQuick(int[] myArray) : base(myArray)
+        {
+        }
+        public void Swap(ref int x, ref int y)
```

Comment:
it should be private
you use it only in this class, it is a helper function
should be available in another class